coordinator/src/db/hodl_invoice.rs

```diff
@@ -69,6 +69,16 @@ pub fn get_r_hash_by_order_id(conn: &mut PgConnection, order_id: Uuid) -> QueryR
         .get_result(conn)
 }
 
+pub fn get_pre_image_by_order_id(
```

Comment:
Uh, the API confused me. It returns `QueryResult<Option<String>>` which I would expect it means that if no pre-image was found, it returns `None`, however, the `get_result` will return an `Err(NotFound)` if it no element was found (by `order_id`) but it will return `None` if the hodl_invoice's pre-image was null...

I had to think unnecessarily long about this... maybe a rust doc could help? :D